lib/services/search_service.dart

```diff
@@ -388,6 +390,27 @@ class SearchService {
   final _causeServiceClient = CauseApiClient();
   final _causesService = locator<CausesService>();
 
+  var _random = Random();
+
+  void updateShuffleSeed() {
+	_random = Random();
+  }
+
+  List<T> _shuffleList<T>(List<T> list) {
+	final newList = List<T>.from(list);
+	newList.shuffle(_random);
+	return newList;
+  }
+
+  List<T> _orderSearchResults<T>(List<T> list, SearchQuery? query) {
+    // If the list has already been sorted by query then don't change
+	if (query?.sort != null) {
+	  return list;
+	}
+
+	return _shuffleList(list);
```

Comment:
Looks like the indents in this file is broken 🤔 Simple autoformat will do the thing I believe. I've got it turned on automatically before the commit, highly recommend 😄